app/api/resources/templates/resolver/resolver_id_content.html

```diff
@@ -0,0 +1,79 @@
+{% import "shared/url.html" as url_helper %}
+
+<div class="redirect-content">
+    <div class="redirect-info">
+        <div class="search-param">
+            <span class="search-param-label">Gene</span>
+            <span class="search-param-value">{{ response.stable_id }}</span>
+        </div>
+        {% if response.code == 404 and response.message %}
+            <div class="error-message">
+                {{ response.message }}
+            </div>
+            {{ url_helper.render_url("Go to", "https://beta.ensembl.org/species-selector") }}
+        {% else %}
+            {% for item in response.content %}
+                <div class="id-detail-container">
+                    <span class="id-detail">{{ item.common_name }},</span>
+                    <span class="id-detail id-detail-margin-left id-detail-italic">{{ item.scientific_name }},</span>
+                    <span class="id-detail id-detail-margin-left id-detail-italic">
+                        {% if item.type and item.is_reference %}
+                          {{ item.type.kind }} - {{ item.type.value }}, Reference
+                        {% elif item.is_reference %}
+                          Reference
+                        {% elif item.type %}
+                          {{ item.type.kind }} - {{ item.type.value }}
+                        {% else %}
+                          -
+                        {% endif %}
```

Comment:
I don't think we want the hyphen. Hyphens make sense in a table, to fill an otherwise empty cell; but if what you are displaying isn't a table, then a hyphen is misleading. Also, if we don't display anything here, then that extra comma after this block shouldn't be displayed either.